std/digest/digest.d

```diff
@@ -991,3 +991,21 @@ unittest
     //necessary, result will have the correct length, but buf will still have it's original
     //length
 }
+
+unittest
```

Comment:
OK, I didn't know that, but:

> A unittest that is deliberately unsafe should be annotated with `@system`. Meaning that any un-annotated unittest needs corrective action one way or the other.

This test is not deliberately unsafe: It's blocked by `toHexString` being system. So it's probably better to leave this as is and wait till somebody fixes `toHexString`.